src/styles/starsreviews.css

```diff
@@ -0,0 +1,34 @@
+/* #region starsreviews */
+.product-card__reviews-container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.stars {
+  display: flex;
+  margin-top: 16px;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
```

Comment:
Please remove margin for last star
![image](https://github.com/mate-academy/layout_product-cards/assets/54936044/c25e93c3-9748-4822-b615-26abdbb73f58)